library/script/bpl_libraries.py

```diff
@@ -18,6 +18,7 @@ def ingest(self) -> pd.DataFrame:
         for i in content["locations"]:
             data.append(i["data"])
         df = pd.DataFrame.from_dict(data, orient="columns")
+        df.loc[df["position"] == "", "position"] = ","
```

Comment:
I am wondering if it is okay to just drop the record entirely. I looked at the csv that it is generated from this script and the address for the record without a lat/long is a duplicate of another record 
<img width="792" alt="Screenshot 2022-12-14 at 12 08 11 PM" src="https://user-images.githubusercontent.com/41444592/207661145-600070bb-3200-4508-ad91-5651b2226d2a.png">

Reply:
although I guess it doesn't really matter, the hacky solution works as well

Reply:
ah! That's a good observation. I don't have super strong feeling other than if we ever got similar case again it would be good to handle them in general as well?

Reply:
Yeah I guess my only thought is I wonder how this might affect facilities downstream (?). I think your solution works and is probably the best "general" way of handling these exceptions